treeherder/config/settings.py

```diff
@@ -320,6 +321,13 @@
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
 }
 
+# User agents which will be blocked from making requests to the site.
+DISALLOWED_USER_AGENTS = (
+    re.compile(r'^libcurl/'),
+    re.compile(r'^Python-urllib/'),
+    re.compile(r'^python-requests/'),
+)
+
```

Comment:
I think we should only disable these on production, could you add a setting for that?